clients/da-vinci-client/src/main/java/com/linkedin/davinci/VersionBackend.java

```diff
@@ -325,8 +342,11 @@ synchronized CompletableFuture<Void> subscribe(ComplementSet<Integer> partitions
         partitionFutures.computeIfAbsent(partition, k -> CompletableFuture.completedFuture(null));
       } else {
         partitionFutures.computeIfAbsent(partition, k -> new CompletableFuture<>());
-        // AtomicReference of storage engine will be updated internally.
-        backend.getIngestionBackend().startConsumption(config, partition);
+        try {
+          dbBootstrapper.bootstrapDatabase(storeName, versionNumber, partition);
+        } catch (Exception e) {
```

Comment:
What do you expect to do when exception is caught?

Reply:
`bootstrapDatabase` attempts to bootstrap from blobs and if there's an exception at that point, it falls back to kafka. It looks like StoreIngestions handles other exceptions so I am just logging here. Is there anything else we should be doing here?